### PR TITLE
make change to 2nd case so that it publishes to topic and add std Con…

### PR DESCRIPTION
**Pull request by phderome**
…sumerConfig AUTO_OFFSET_RESET_CONFIG (earliest) setting and a bit of usage comments.

### CODE REVIEW

**Comment by phderome** [comment]
Not sure where all the errors come from as I am not that familiar as to how to integrate kafka-examples with full Kafka dependencies properly. In event, mvn -DskipTests=true package worked fine in my local copy.

**Comment by miguno** [comment]
Thanks for this PR, @phderome!

For legal reasons we need you to sign the Contributor License Agreement (see above), sorry for having to ask for this.

File: kafka-streams/src/main/java/io/confluent/examples/streams/MapFunctionLambdaExample.java [hunk 1]
```diff
@@ -32,6 +33,17 @@
  * Use cases include e.g. basic data sanitization, data anonymization by obfuscating sensitive data
  * fields (such as personally identifiable information aka PII).
  *
+ * ./bin/kafka-console-consumer --zookeeper localhost --topic UppercasedTextLinesTopic
```
Comment (miguno) [inline kafka-streams/src/main/java/io/confluent/examples/streams/MapFunctionLambdaExample.java]: What about also explaining how to write some example input data via `./bin/kafka-console-producer` while we're at it?

**Comment by phderome** [comment]
i was trying to sign it at the time but had usage difficulty. In all
honesty, I think Damian and team will have things very well covered and may
not find this PR all that useful. I had initiated it prior to your
commitment for fixes in short term.

So, I think we can just let this PR die.

**Comment by miguno** [comment]
@phderome : Ok, still leave it open for the moment.  Also:  The test failures above are not related to your code changes (i.e. no mistake on your part).

**Comment by phderome** [comment]
I just signed as individual contributor.


Nice to hear about no mistakes on my part, ok.

**Comment by miguno** [comment]
Closed because the MapLambdaFunctionExample has already been improved (with even more detailed step-by-step instructions).

**Pull request closed without merging**